hack/quick-install.sh

```diff
@@ -67,8 +69,8 @@ apply() {
     --set nodeExporter.enabled=false \
     --set prometheus.enabled=false
 
-  helm upgrade --install karpenter charts/karpenter
+  helm upgrade --install karpenter karpenter/karpenter
 }
 
 usage
-main "$@"
+main "$@"
```

Comment:
Can you leave the trailing newline here?

Reply:
No problem, will do.